packages/journey-manager/src/index.ts

```diff
@@ -209,14 +210,52 @@ function filterMap<X, Y>(
   }, []);
 }
 
+const resolveTriggers = async (
+  config: Config,
+  triggers?: Triggers,
+): Promise<Triggers> => {
+  if (triggers != null) {
+    return triggers;
+  }
+  const baseUrl =
+    config.dev ?? false
+      ? "https://triggers.dev.nlx.ai"
+      : "https://triggers.nlx.ai";
```

Comment:
It's `https://triggers.dev.mm.nlx.ai` and `https://triggers.mm.nlx.ai`

Reply:
@vlad31 oh good point, updated!